src/components/create-hack.js

```diff
@@ -97,6 +99,12 @@ export function CreateHack(props) {
               <Grid className={classes.field} item xs={12}>
                 <Typography variant="h4">New Hack Idea</Typography>
               </Grid>
+              <Grid className={classes.field} item xs={12}>
+                <EditHackStatus
+                  status=""
+                  handleUpdate={(statusValue) => setStatus(statusValue)}
```

Comment:
handleUpdate can just be set to handleUpdate={setStatus} the rest is handled by the component